test/specs/browser/riot/core.spec.js

```diff
@@ -43,6 +43,7 @@ import '../../../tag/update-context.tag'
 import '../../../tag/dynamic-virtual.tag'
 import '../../../tag/multiple-select.tag'
 import '../../../tag/dynamic-nested.tag'
+import '../../../tag/dont-remove-attr-with-empty-string-vals.tag'
```

Comment:
please use a file name that represents also your tag id for example `bug-2629.tag`